extractor/src/test/java/org/schabi/newpipe/extractor/services/peertube/PeertubeChannelLinkHandlerFactoryTest.java

```diff
@@ -30,7 +30,7 @@ public void acceptUrlTest() throws ParsingException {
 
     @Test
     public void getIdFromUrl() throws ParsingException {
-        assertEquals("kranti@videos.squat.net", linkHandler.fromUrl("https://peertube.mastodon.host/accounts/kranti@videos.squat.net").getId());
-        assertEquals("kranti@videos.squat.net", linkHandler.fromUrl("https://peertube.mastodon.host/accounts/kranti@videos.squat.net/videos").getId());
+        assertEquals("accounts/kranti@videos.squat.net", linkHandler.fromUrl("https://peertube.mastodon.host/accounts/kranti@videos.squat.net").getId());
+        assertEquals("accounts/kranti@videos.squat.net", linkHandler.fromUrl("https://peertube.mastodon.host/accounts/kranti@videos.squat.net/videos").getId());
     }
```

Comment:
Please also add a test for getting an URL from an ID. It should test all of: `whatever`, `video-channels/whatever`, and `accounts/whatever`. Obviously don't use `whatever`, but actually existing things.